modules/tinymce/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Inline toolbars flickered when switching between editors #TINY-8594
 - Multiple inline toolbars were shown if focused too quickly #TINY-8503
 - Added background and additional spacing for the text labeled buttons in the toolbar to improve visual clarity #TINY-8617
+- Fix custom toolbar select buttons width on touch devices #TINY-8647
```

Comment:
```suggestion
- Toolbar split buttons with text used an incorrect width on touch devices #TINY-8647
```

Reply:
Accepting this sorry Alex, as we'd like to get ready for the release. If there's any issues with it let us know and we'll make sure to correct it before we actually release.